code/controllers/subsystem/mapping.dm

```diff
@@ -96,9 +100,34 @@ SUBSYSTEM_DEF(mapping)
    Used by the AI doomsday and the self destruct nuke.
 */
 
-/datum/controller/subsystem/mapping/fire()
-	if(clearing_reserved_areas)
-		wipe_turf_reservations()
+/datum/controller/subsystem/mapping/proc/wipe_reservations(wipe_safety_delay = 100)
+	if(clearing_reserved_turfs || !initialized)			//in either case this is just not needed.
+		return
+	clearing_reserved_turfs = TRUE
+	SSshuttle.transit_requesters.Cut()
+	message_admins("Clearing dynamic reservation space.")
+	var/list/obj/docking_port/mobile/in_transit = list()
+	for(var/i in SSshuttle.transit)
+		var/obj/docking_port/stationary/transit/T = i
+		if(!istype(T))
+			continue
+		in_transit[T] = T.get_docked()
+	var/go_ahead = world.time + wipe_safety_delay
+	if(in_transit.len)
+		message_admins("Shuttles in transit detected. Attempting to fast travel. Timeout is [go_ahead/10] seconds.")
```

Comment:
Should be `wipe_safety_delay`, as is it just shows duration since roundstart since `world.time` is included.

Reply:
OOF